README.md

```diff
@@ -76,11 +79,11 @@ See usage:
 2. Run the CloudFormation template and creat the deployment, wait till the stack creation completes.
 3. Extract the following using the AWS CLI.
    * Bastion node public IP. (Used as the JMeter client)
-   * Load balancer hostname.
-   * Private IPs of WSO2 IS instances.
+   * Private IP of WSO2 IS instance.
    * RDS instance hostname.
-4. Copy `setup-bastion.sh`, key file and the JMeter setup to bastion node.
-5. SSH into the bastion node and execute the copied [setup-bastion.sh](distribution/scripts/setup) script, which will setup the additional components in the deployment.
-6. SSH into the bastion node and execute the [run-performance-test.sh](distribution/scripts/jmeter) script, which will run the test and collect the results.
-7. Download the test results from the bastion node.
-8. Create the `summary.csv` file and the `summary.md` file. 
+4. Setup the wso2 IS server in the instance and create the databases
+5. Copy `setup-bastion.sh`, key file and the JMeter setup to bastion node.
```

Comment:
I thought we are coping the whole target tar ball?

Reply:
resolved